legal-api/src/legal_api/services/filings/validations/dissolution.py

```diff
@@ -12,17 +12,37 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Validation for the Voluntary Dissolution filing."""
+from enum import Enum
 from http import HTTPStatus
-from typing import Dict, Optional
+from typing import Dict, Final, Optional
 
+import pycountry
 from flask_babel import _
 
 from legal_api.errors import Error
-from legal_api.models import Business
+from legal_api.models import Address, Business, PartyRole
 
 from ...utils import get_str
 # noqa: I003; needed as the linter gets confused from the babel override above.
 
+CORP_TYPES: Final = [Business.LegalTypes.COMP.value,
+                     Business.LegalTypes.BCOMP.value,
+                     Business.LegalTypes.CCC_CONTINUE_IN.value,
+                     Business.LegalTypes.BC_ULC_COMPANY.value]
+
+
+class DissolutionTypes(Enum):
```

Comment:
```suggestion
class DissolutionTypes(str, Enum):
```
Allows the Enum to be comparable to string without referencing the ```.value```